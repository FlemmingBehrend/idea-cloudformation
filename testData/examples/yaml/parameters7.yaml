AWSTemplateFormatVersion: '2010-09-09'
Parameters:
  MyNestedStack:
    Type: String
    NoEcho: a
Resources:
  RootBucket:
    Type: AWS::S3::Bucket
