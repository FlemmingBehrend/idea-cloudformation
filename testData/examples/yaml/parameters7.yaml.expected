AWSTemplateFormatVersion: '2010-09-09'
Parameters:
  MyNestedStack:
    Type: String
    0@<NoEcho:> a
Resources:
  RootBucket:
    Type: AWS::S3::Bucket

0: Only 'True' value is allowed
